### PR TITLE
V1.4.0

### PR DESCRIPTION
**Pull request by generall**

### CODE REVIEW

File: tests/congruence_tests/test_collections.py [hunk 1]
```diff
@@ -42,3 +45,93 @@ def test_get_collection():
...
+            ),
+        ),
+    )
+
+    local_collection_info = local_client.get_collection(COLLECTION_NAME)
+    remote_collection_info = remote_client.get_collection(COLLECTION_NAME)
+
+    assert local_collection_info.config.params.vectors["text"].hnsw_config is None
+    assert local_collection_info.config.params.vectors["text"].quantization_config is None
+    assert local_collection_info.config.hnsw_config.ef_construct == 123
```
Comment (timvisee) [inline tests/congruence_tests/test_collections.py:126]: In the local client `ef_construct` is set to 123 in an `update_collection` request, but it remains 100 which was the default.

Is this a problem or intended behavior for the local client?

The remote client works as expected.

**Review by timvisee — COMMENTED** [review]
I added a simple test for the new update collection parameters feature.

I did find a potential problem with the local client, please see the comment below.

File: tests/congruence_tests/test_collections.py (hunk 1 above)
Comment (generall) [reply tests/congruence_tests/test_collections.py:126]: collection params doesn't have an effect in local client

**Review by timvisee — APPROVED** [review]
Good to see the tests succeed now :raised_hands: 

Take my :heavy_check_mark: with a grain of salt, I'm no Python wizard.

**Pull request merged**